internal/cli/atlas_dbusers_delete.go

```diff
@@ -70,6 +70,7 @@ func AtlasDBUsersDeleteBuilder() *cobra.Command {
 	cmd.Flags().BoolVar(&opts.confirm, flags.Force, false, usage.Force)
 
 	cmd.Flags().StringVar(&opts.projectID, flags.ProjectID, "", usage.ProjectID)
+	cmd.Flags().StringVar(&opts.authDB, flags.AuthDB, "", usage.AuthDB)
```

Comment:
let's make this retro-compatible, this means we don't require it and we default to "admin"
```suggestion
	cmd.Flags().StringVar(&opts.authDB, flags.AuthDB, "admin", usage.AuthDB)
```